raiden/network/rpc/client.py

```diff
@@ -680,7 +680,7 @@ def eth_estimateGas(
             to: address = b'',
             value: int = 0,
             data: bytes = b'',
-            startgas: Optional[int] = None) -> int:
+            startgas: Optional[int] = None) -> Optional[int]:
```

Comment:
btw, you don't need to declare `Optional[int]` if the default value is `None`, it's inferred for you:

`startgas: Optional[int] = None) -> Optional[int]:`

is the same as:

`startgas: int = None) -> Optional[int]:`

Additionally, please use the format we aggreed on:

```
        startgas: Optional[int] = None
) -> Optional[int]:
```

Reply:
Nice, didn't know about that.

Reply:
Me neither, I literraly learned it half an hour ago :) 

https://www.python.org/dev/peps/pep-0484/#union-types

Reply:
That's cool!